src/components/SnartSluttPaSykepengene/MaxDateIngress.tsx

```diff
@@ -0,0 +1,83 @@
+import { Alert, BodyLong, Box, Heading, Skeleton } from '@navikt/ds-react'
+import { ReactElement } from 'react'
+import Link from 'next/link'
+
+import { useLogAmplitudeEvent } from '@/libs/amplitude/amplitude'
+import { trpc } from '@/utils/trpc'
+
+function MaxDateErrorMessage({ reason }: { reason: string }): ReactElement {
+  useLogAmplitudeEvent(
+    {
+      eventName: 'alert vist',
+      data: { variant: 'error', tekst: 'Beklager, teknisk feil med maks dato' },
+    },
+    { grunn: reason },
+  )
+
+  return (
+    <Alert variant="error" className="mb-8">
+      <Heading size="medium" spacing level="1">
+        Beklager, teknisk feil
+      </Heading>
+      <BodyLong>På grunn av feil i systemene våre får vi ikke vist deg beregnet siste dag med sykepenger.</BodyLong>
+      <BodyLong spacing>Vennligst prøv igjen litt senere.</BodyLong>
+      <BodyLong>
+        <Link href="https://www.nav.no/no/nav-og-samfunn/kontakt-nav/teknisk-brukerstotte/kontakt-teknisk-brukerstotte-nav.no">
+          Kontakt teknisk brukerstøtte dersom problemene vedvarer.
+        </Link>
+      </BodyLong>
+    </Alert>
+  )
+}
+
+function MaxDateContent(): ReactElement {
+  const maxDate = trpc.maxDate.useQuery()
+
+  switch (maxDate.status) {
+    case 'error':
+      return <MaxDateErrorMessage reason="Api error" />
+    case 'loading':
+      return (
+        <Box>
+          <Skeleton variant="text" height="7rem" />
+          <Skeleton variant="text" height="4rem" />
+        </Box>
+      )
+    case 'success':
+      if (maxDate.data.maxDate && maxDate.data.utbetaltTom) {
+        return (
+          <BodyLong size="medium" spacing>
+            Per {maxDate.data.utbetaltTom} er det beregnet at din siste dag med sykepenger er {maxDate.data.maxDate}.
+            Datoen gjelder hvis du er sykmeldt uten opphold. Den vil flytte seg hvis du for eksempel jobber noen
+            perioder, eller hvis du tar ferie. Du kan få sykepenger i maksimalt 52 uker, og for deg vil det si at denne
+            datoen nå nærmer seg.
+          </BodyLong>
+        )
+      } else {
+        return (
+          <MaxDateErrorMessage
+            reason={`Missing: ${maxDate.data.maxDate ? 'maks dato' : ''} ${
+              maxDate.data.utbetaltTom ? 'utbetalt Tom' : ''
+            }`}
+          />
```

Comment:
Hvordan ser det ut i gui hvis dette inntreffer?

Reply:
![Screenshot 2023-12-11 at 11 44 13](https://github.com/navikt/meroppfolging-frontend/assets/8613937/130035e4-281b-454f-8598-7068fee29919)


Den prop'en blir bare brukt av amplitude for å "logge" hvilket felt som mangler.

Reply:
Ah, ok, skjønner

Reply:
Jeg ser nå at jeg returener feil verdi i ternary 🙈

Reply:
Haha, ja, det la jeg ikke merke til.